53.md

```diff
@@ -84,6 +84,79 @@ Event `kind:1311` is live chat's channel message. Clients MUST include the `a` t
 
 Hosts may choose to pin one or more live chat messages by updating the `pinned` tags in the live event kind `30311`.
 
+### Stream Raids
+
+Event `kind:1312` is a live stream raid (redirect viewers to another stream).
+
+Clients MUST include two `a` tags, one marked as `from` and the other marked as `to` which redirects viewers to a new live stream.
+The `content` MAY contain a raid message.
+
+Clients SHOULD automatically redirect viewers from one stream to another. 
+This is a similar function used on Twitch also called raids.
+
+```jsonc
+{
+  "kind": 1312,
+  "tags": [
+    ["a", "30311:<Community event author pubkey>:<d-identifier of the community>", "<Optional relay url>", "from"],
+    ["a", "30311:<Community event author pubkey>:<d-identifier of the community>", "<Optional relay url>", "to"],
```

Comment:
IMO using a different single letter tag is a better solution than using a "label" at the end of the tag. things always seem to get messy when arbitrary strings are added to tags

Reply:
Ok how about just using the order then?